### PR TITLE
Ap 1901 policy disregards

### PR DESCRIPTION
**Pull request by eloisetait3**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-1901)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- [x] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/helpers/check_answers_helper.rb [hunk 1]
```diff
@@ -72,4 +72,21 @@ def check_long_question_no_link(question:, answer:, name:, no_border: false)
   def number_to_currency_or_na(number)
     number.to_d == 999_999_999_999.0.to_d ? 'N/a' : gds_number_to_currency(number)
   end
+
+  def safe_yes_or_no(value)
```
Comment (Alex-Swann) [inline app/helpers/check_answers_helper.rb:76]:
```
def safe_yes_or_no(value)
  return unless !!value == value
  value ?  'Yes' : 'No'
end
```
Can be refactored even if you want to keep `boolean?` for clarity

File: app/helpers/policy_disregards_helper.rb [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    criminal_injuries_compensation_scheme
+    national_emergencies_trust
+    we_love_manchester_emergency_fund
+    london_emergencies_trust
+  ].freeze
+
+  def policy_disregards_hash(policy_disregards)
+    items = policy_disregards_as_array(policy_disregards)
+    items&.compact!
+    return nil unless items.present?
```
Comment (Alex-Swann) [inline app/helpers/policy_disregards_helper.rb:17]: `return nil unless items.present?` 
can be
`return unless items.present?` which returns nil without needing to specify it

File: app/helpers/policy_disregards_helper.rb [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+    {
+      items: items
+    }
+  end
+
+  def policy_disregards_as_array(policy_disregards)
+    ATTRIBUTES&.map do |attribute|
```
Comment (Alex-Swann) [inline app/helpers/policy_disregards_helper.rb:25]: why is the `&` needed here?

File: app/models/legal_aid_application.rb [hunk 4]
```diff
@@ -8,6 +8,7 @@ class LegalAidApplication < ApplicationRecord
   SECURE_ID_DAYS_TO_EXPIRE = 7
   WORKING_DAYS_TO_COMPLETE_SUBSTANTIVE_APPLICATION = 20
   CCMS_SUBMITTED_STATES = %w[generating_reports submitting_assessment assessment_submitted].freeze
+  POLICY_DISREGARDS_START_DATE = Date.new(2021, 1, 8)
```
Comment (Alex-Swann) [inline app/models/legal_aid_application.rb]: 💥

File: spec/helpers/policy_disregards_helper_spec.rb [hunk 5]
```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe PolicyDisregardsHelper, type: :helper do
+  include ApplicationHelper
+
+  describe '#policy_disregards_hash' do
+    # subject(:policy_disregards_helper) { policy_disregards_hash(policy_disregards) }
+
+    # it { is_expected.to be_a(Hash) }
```
Comment (Alex-Swann) [inline spec/helpers/policy_disregards_helper_spec.rb:9]: Remove comments

**Review by Alex-Swann — APPROVED** [review]
Approved pending very minor remarks

File: app/helpers/policy_disregards_helper.rb (hunk 3 above)
Comment (eloisetait3) [reply app/helpers/policy_disregards_helper.rb:25]: It's syntax to make sure there are values there before mapping over (Colin says it's called the safe navigation operator)

Comment (Alex-Swann) [reply app/helpers/policy_disregards_helper.rb:25]: Ah cool, I know it's used for digging, but good to know 👍

Comment (stephenrichards) [reply app/helpers/policy_disregards_helper.rb:25]: But if it's a constant, I think you can be pretty sure it's there, no?

**Pull request merged**